applications/jupyter-lab-hpc-native/app.json

```
@@ -0,0 +1,82 @@
+{
+    "id": "jupyter-lab-hpc-native",
+    "version": "1.0.2",
+    "description": "Run an interactive Jupyter Notebook session with optional MPI support.",
+    "owner": "${apiUserId}",
+    "enabled": true,
+    "runtime": "ZIP",
+    "containerImage": "tapis://cloud.data/corral/tacc/aci/CEP/applications/v3/jupyter-lab-hpc-native/jupyter-lab-hpc-native.zip",
+    "jobType": "BATCH",
+    "maxJobs": -1,
+    "maxJobsPerUser": -1,
+    "strictFileInputs": true,
+    "jobAttributes": {
+        "description": null,
+        "dynamicExecSystem": false,
+        "execSystemId": "frontera",
+        "execSystemExecDir": "${JobWorkingDir}",
+        "execSystemOutputDir": "${JobWorkingDir}/output",
+        "execSystemLogicalQueue": "development",
+        "archiveSystemId": "cloud.data",
+        "archiveSystemDir": "HOST_EVAL($HOME)/tapis-jobs-archive/${JobCreateDate}/${JobName}-${JobUUID}",
+        "archiveOnAppError": true,
+        "isMpi": false,
+        "cmdPrefix": null,
+        "parameterSet": {
+            "appArgs": [
+                {
+                    "name": "launchParameters",
+                    "arg": "echo 'Job is done.'",
+                    "inputMode": "FIXED",
+                    "notes": {
+                        "isHidden": true
+                    }
+                }
+            ],
+            "containerArgs": [],
+            "schedulerOptions": [
+                {
+                    "name": "TACC Scheduler Profile",
+                    "description": "Scheduler profile for HPC clusters at TACC",
+                    "inputMode": "FIXED",
+                    "arg": "--tapis-profile python3-3.9.2",
+                    "notes": {
+                        "isHidden": true
+                    }
+                },
+                {
+                    "name": "TAP Session Substring",
+                    "description": "TAP Functions require the substring 'tap_' in the SLURM job name.",
+                    "inputMode": "FIXED",
+                    "arg": "--job-name ${JobName}-tap_",
```

let's include `dsmydata` to mount ds my data
```suggestion
                    "arg": "--job-name ${JobName}-tap_dsmydata",
```